packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -5,38 +5,106 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, wire, track, api } from 'lwc';
-import { productsByQuery } from 'commerce/data';
+// import { productsByQuery } from 'commerce/data';
 
-//
+import { useQuery } from '@lwce/apollo-client';
+import gql from 'graphql-tag';
+import { apiClient } from '../api/client';
+
+// const QUERY = gql``;
+const QUERY = gql`
```

Comment:
A large query like this could be in a separate file and imported. That would also make it reusable. But I do know that we want to maybe use gql `fragments` in the future spread across the sub lwc components.